deploy/playbooks/roles/common/tasks/postgresql.yml

```diff
@@ -9,7 +9,7 @@
   sudo: yes
   lineinfile:
      # TODO: should not hardcode that version
-     dest: /etc/postgresql/9.3/main/pg_hba.conf
+     dest: /etc/postgresql/9.5/main/pg_hba.conf
```

Comment:
Seems wrong that we're changing to a generic postgresql package dependency but keeping a specific (and different) version in this path.  I see the TODO, but, isn't there either a generic link without version or a way to query the installed version?